include/unknown_key.h

```diff
@@ -33,7 +33,7 @@ namespace pgp {
              *  Constructor
              */
             template <class decoder, class = std::enable_if_t<is_decoder_v<decoder>>>
-            unknown_key(decoder&) noexcept {}
+            explicit unknown_key(decoder&) noexcept {}
```

Comment:
This function should be defaulted.

Reply:
Defaulting this generates an error:

    error: only special member functions may be defaulted